pkg/packetfilter/iptables/iptables.go

```diff
@@ -52,7 +54,8 @@ var (
 )
 
 type PacketFilter struct {
-	ipt *iptables.IPTables
+	ipt        *iptables.IPTables
+	IPSetIface ipset.Interface
```

Comment:
Let's not export this.

```suggestion
	ipSetIface ipset.Interface
```